src/utils/layerUtils.js

```diff
@@ -46,14 +46,22 @@ export function getTimestampFromConfig(config, previewYear, isTimeSliderActive)
 /**
  * @param {GeoAdminWMTSLayer | ExternalWMTSLayer} wmtsLayerConfig
  * @param {CoordinateSystem} projection
- * @param {Number} previewYear
- * @param {Boolean} isTimeSliderActive
+ * @param {Boolean} [options.addTimestamp=false] Add the timestamp from the time config or the
+ *   timeslider to the ur. When false the timestamp is set to `{Time}` and need to processed later
+ *   on. Default is `false`
+ * @param {Number | null} [options.previewYear=null] Default is `null`
+ * @param {Boolean | null} [options.isTimeSliderActive=null] Default is `null`
  * @returns {String | null}
  */
-export function getWmtsXyzUrl(wmtsLayerConfig, projection, previewYear, isTimeSliderActive) {
+export function getWmtsXyzUrl(wmtsLayerConfig, projection, options = {}) {
+    const { addTimestamp = false, previewYear = null, isTimeSliderActive = null } = options ?? {}
```

Comment:
`isTimeSliderActive` default value is `false`. I would keep consistency here and put `false` as the default value.